release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -326,6 +326,21 @@ Admin Network Policy is available as a Technology Preview feature.
 
 You can enable `AdminNetworkPolicy` and `BaselineAdminNetworkPolicy` resources, which are part of the Network Policy V2 API, in clusters running the OVN-Kubernetes CNI plugin. Cluster administrators can apply cluster-scoped policies and safeguards for an entire cluster before namespaces are created. Network administrators can secure clusters by enforcing network traffic controls that cannot be overridden by users. Network administrators can enforce optional baseline network traffic controls that can be overridden by users in the cluster, if necessary. Currently, these APIs support only expressing policies for intra-cluster traffic.
 
+[id="ocp-4-14-creating-subinterface"]
+==== MAC-VLAN, IP-VLAN, and VLAN subinterface creation for pods
+
+With this release, the ability to create a MAC-VLAN, IP-VLAN, and VLAN subinterface based on a master interface in a container namespace is generally available. This feature lets you create master interfaces within the pod network configuration using a separate network attachment definition. You can then base the VLAN, MACVLAN or IPVLAN on this interface without knowing the network configuration of the node. For more information, see xref:../networking/multiple_networks/configuring-additional-network.html#nw-about-configuring-master-interface-container_configuring-additional-network[About configuring the master interface in the container network namespace].
```

Comment:
```suggestion
With this release, the ability to create a MAC-VLAN, IP-VLAN, and VLAN subinterface based on a master interface in a container namespace is generally available. This feature lets you create master interfaces within the pod network configuration using a separate network attachment definition. You can then base the VLAN, MACVLAN or IPVLAN on this interface without knowing the network configuration of the node. For more information, see xref:../networking/multiple_networks/configuring-additional-network.html#nw-about-configuring-master-interface-container_configuring-additional-network[About configuring the master interface in the container network namespace].
```
Still not a fan of the "This feature lets you create" construction, which goes explicitly against the ISG guidance against anthropomorphism: https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism. Is there a reason why this can't be revised so that "feature" isn't the main agent of action here and the user is?

Reply:
OK I resolved that thanks @bburt-rh